decisions.md

```diff
@@ -0,0 +1,20 @@
+## The Good Docs Project Decisions
+
+### Criteria for Decision Making
+
+Decisions are based on voting using the following:
+
+    +1 : For
+    +0: Mildly for, but mostly indifferent
+    0: Indifferent
+    -0: Mildly against, but mostly indifferent
+    -1 : Strongly against. This is expected to be accompanied by a reason.
+
+For most decisions, a motion is raised within one of TheGoodDocsProject open forums (slack, email list, etc) and those present vote and decide. Should a decision:
+
+    Be strategic or impact many within TheGoodDocsProject,
+    Or if there is contention in deciding, and after discussion at least one member is still strongly against the motion, by continuing to vote -1,
+
+Then the motion should be referred to the PSC.
```

Comment:
```suggestion
For most decisions, a motion is raised within one of TheGoodDocsProject open forums (slack, email list, etc) and those present vote and decide. A decision should be referred to the PSC only if:

- The decision is strategic or could impact many people within TheGoodDocsProject
- If there is contention in deciding, and after discussion at least one member is still strongly against the motion, by continuing to vote -1
```

Reply:
We don't need to worry about fixing this lift-and-shift of the decisions page in this iteration.
It is about to get updated again based on https://docs.google.com/document/d/1hwdA2Gh3ePUFXk4juu7mSoAraxV-BDiarXaPCOY6K3w/edit

Reply:
Noted.